modules/exploits/linux/http/rconfig_ajaxarchivefiles_rce.rb

```diff
@@ -0,0 +1,199 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'Rconfig 3.x Chained Remote Code Execution',
```

Comment:
Hey - thanks for the PR :+1: There's some formatting issues in this file - would you mind running Rubocop on it? That's our automated code quality tool. You should be able to run it with:

```
rubocop -a modules/exploits/linux/http/rconfig_ajaxarchivefiles_rce.rb 
```

Let me know if you have any issues

Reply:
Hi !
I will run it and commit changes.